apps/api/e2e/tests/admin.test.ts

```diff
@@ -755,6 +755,74 @@ describe("Admin API", () => {
     await adminClient.deleteAgent(user3Result.agent!.id);
   });
 
+  test("should use 'join' with different query param formats", async () => {
```

Comment:
@brunocalza alrighty, dropped the `searchType` and replaced it with `join`. the different flows are:
1. no `join` => `false`
2. `join=false` => `false`
2. `join=true` => `true`
3. `join` => `true` (aka naked `join` is allowed)